app/controllers/users_controller.rb

```diff
@@ -1,4 +1,12 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, except: %i(show new create)
+  before_action :correct_user, only: %i(edit update)
+  before_action :admin_user, only: :destroy
+
+  def index
+    @users = User.page(params[:page])
```

Comment:
```suggestion
    @users = User.page(params[:page]).per <number>
```
Dont forget to move number to settings file